scipy/stats/_stats_py.py

```diff
@@ -1013,6 +1013,16 @@ def _moment(a, moment, axis, *, mean=None):
         # Starting point for exponentiation by squares
         mean = a.mean(axis, keepdims=True) if mean is None else mean
         a_zero_mean = a - mean
+
+        eps = np.finfo(a_zero_mean.dtype).resolution * 1e6
+        rel_diff = np.max(np.abs(a_zero_mean), axis=axis, keepdims=True)/mean
+        with np.errstate(invalid='ignore'):
+            precision_loss = np.any((0 < rel_diff) & (rel_diff < eps))
```

Comment:
```suggestion
            precision_loss = np.any(rel_diff < eps)
```
We probably _do_ want this warning to show when all the data is exactly numerically equal.

Reply:
> We probably do want this warning to show when all the data is exactly numerically equal.

Yes, agreed.

> I don't think it was correct in [gh-13254](https://github.com/scipy/scipy/pull/13254) to set the `skew`/`kurtosis` to zero when all data were equal; it should be `nan`, and there should be a warning.

I agree: 0 is not the correct skew/kurtosis for near-constant inputs.